src/Xamarin.Android.Build.Tasks/Tasks/CollectNativeFilesForArchive.cs

```diff
@@ -0,0 +1,325 @@
+#nullable enable
+// Copyright (C) 2011 Xamarin, Inc. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+
+using Microsoft.Build.Utilities;
+using Microsoft.Build.Framework;
+
+using ArchiveFileList = System.Collections.Generic.List<(string filePath, string archivePath)>;
+using Microsoft.Android.Build.Tasks;
+
+namespace Xamarin.Android.Tasks;
+
+public class CollectNativeFilesForArchive : AndroidTask
```

Comment:
The other ones had a summary, might be worth putting a quick one here, too.

Reply:
👍